src/dcm_classifier/utility_functions.py

```diff
@@ -627,6 +652,23 @@ def sanitize_dicom_dataset(
     return dataset_dictionary, True
 
 
+def validate_numerical_dataset_element(element: str) -> str | float:
```

Comment:
ensure this function is able to sanitize None Type values as well

Reply:
additionally ansure unit testing for this function with different cases. Be careful using typehints and ensure that function won't crush the tool when encountering wrong type